plugins/node/opentelemetry-instrumentation-knex/src/instrumentation.ts

```diff
@@ -45,8 +59,21 @@ const DEFAULT_CONFIG: KnexInstrumentationConfig = {
 };
 
 export class KnexInstrumentation extends InstrumentationBase<KnexInstrumentationConfig> {
+  private _semconvStability: SemconvStability = SemconvStability.OLD;
+
   constructor(config: KnexInstrumentationConfig = {}) {
     super(PACKAGE_NAME, PACKAGE_VERSION, { ...DEFAULT_CONFIG, ...config });
+
+    for (const entry of getStringListFromEnv('OTEL_SEMCONV_STABILITY_OPT_IN') ??
+      []) {
+      if (entry.toLowerCase() === 'database/dup') {
+        // database/dup takes highest precedence. If it is found, there is no need to read the rest of the list
+        this._semconvStability = SemconvStability.DUPLICATE;
+        break;
+      } else if (entry.toLowerCase() === 'database') {
+        this._semconvStability = SemconvStability.STABLE;
+      }
```

Comment:
I don't see the case where you set the value to `OLD`, but you're checking lower in the code for that case. Wa it suppose to be the default value?

Reply:
the default value is defined in line 62:
```ts
export class KnexInstrumentation extends InstrumentationBase<KnexInstrumentationConfig> {
  private _semconvStability: SemconvStability = SemconvStability.OLD;
}
```